contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -117,6 +117,35 @@ abstract contract VaultHub is AccessControlEnumerableUpgradeable {
         return $.sockets[$.vaultIndex[IHubVault(_vault)]];
     }
 
+    /// @notice Returns all vaults owned by a given address
+    /// @param _owner Address of the owner
+    /// @return An array of vaults owned by the given address
+    function vaultsByOwner(address _owner) external view returns (IHubVault[] memory) {
```

Comment:
Needs coment that this is to be used as view only, not safe for contract intereaction because of cycle